packages/core/src/domain/configuration/configuration.ts

```diff
@@ -144,6 +144,11 @@ export function validateAndBuildConfiguration(initConfiguration: InitConfigurati
     return
   }
 
+  if (initConfiguration.version && typeof initConfiguration.version !== 'string') {
+    display.error('Version must be defined as a string')
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Since we are completely blind on those kind of execution issues, it could worth planning some reflexions around it to see if we could do something better